Makefile

```diff
@@ -121,9 +121,11 @@ deploy: dist
 ifneq ($(and $(MM_SERVICESETTINGS_SITEURL),$(MM_ADMIN_USERNAME),$(MM_ADMIN_PASSWORD),$(CURL)),)
 	@echo "Installing plugin via API"
 	$(eval TOKEN := $(shell curl -i -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/users/login -d '{"login_id": "$(MM_ADMIN_USERNAME)", "password": "$(MM_ADMIN_PASSWORD)"}' | grep Token | cut -f2 -d' ' 2> /dev/null))
-	@curl -s -H "Authorization: Bearer $(TOKEN)" -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins -F "plugin=@dist/$(BUNDLE_NAME)" -F "force=true" > /dev/null && \
-		curl -s -H "Authorization: Bearer $(TOKEN)" -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins/$(PLUGIN_ID)/enable > /dev/null && \
-		echo "OK." || echo "Sorry, something went wrong."
+	$(eval VERSION := $(shell curl -s -H "Authorization: Bearer $(TOKEN)" -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins -F "plugin=@dist/$(BUNDLE_NAME)" -F "force=true" | grep -Po '"version": *\K"[^"]*"' | sed 's/\"//g'))
```

Comment:
Would be great to track down the root issue. Maybe this check helps, 2 notes:
1. 2/5 We should print the 2 different versions in the error message.
2. This doesn't really solve the development case, when the version does not change. Should we (if we don't already) have the build SHA available via the API? (Obviously, as a separate PR)

Reply:
1) commit 27bbdf9 puts them both in.  
2) I do like explicitly handling api calls, though as expressed below...  the entire premise that this PR is based on is an error in setting of an ENV variable.